lua/lvim/plugins.lua

```diff
@@ -113,21 +116,23 @@ local core_plugins = {
       vim.opt.rtp:prepend(path) -- treesitter needs to be before nvim's runtime in rtp
       require("lvim.core.treesitter").setup()
     end,
+    event = "User FileOpened",
+    dependencies = "nvim-ts-context-commentstring",
   },
   {
     "JoosepAlviste/nvim-ts-context-commentstring",
-    event = "VeryLazy",
+    lazy = true,
   },
 
   -- NvimTree
   {
     "kyazdani42/nvim-tree.lua",
-    -- event = "BufWinOpen",
-    -- cmd = "NvimTreeToggle",
     config = function()
       require("lvim.core.nvimtree").setup()
     end,
     enabled = lvim.builtin.nvimtree.active,
+    cmd = { "NvimTreeToggle", "NvimTreeOpen", "NvimTreeFocus" },
```

Comment:
Could we perhaps add NvimTreeFindFileToggle here as well? 
Have been testing this PR today and ran into this limitation with my config as I turn the autochdir equivalent of nvim-tree off